lightkurve/lightcurve.py

```diff
@@ -300,7 +304,8 @@ def append(self, others):
                 new_lc.centroid_col = np.append(new_lc.centroid_col, others[i].centroid_col)
             if hasattr(new_lc, 'centroid_row'):
                 new_lc.centroid_row = np.append(new_lc.centroid_row, others[i].centroid_row)
-        return new_lc
+        if not inplace:
+            return new_lc
```

Comment:
Should we `return new_lc` regardless of whether or not `inplace=True`?  It makes the API more consistent/predictable if we always return something, and it enables chaining.